src/index.ts

```diff
@@ -1,13 +1,33 @@
-import { Application } from 'probot' // eslint-disable-line no-unused-vars
+import bodyParser from 'body-parser'
+import { Request, Response } from 'express' //eslint-disable-line
+import { Application, GitHubAPI } from 'probot' //eslint-disable-line
 
-export = (app: Application) => {
-  app.on('issues.opened', async (context) => {
-    const issueComment = context.issue({ body: 'Thanks for opening this issue!' })
-    await context.github.issues.createComment(issueComment)
-  })
-  // For more information on building apps:
-  // https://probot.github.io/docs/
+export = async (app: Application) => {
+  const router = app.route('/make-in-india-bot')
+
+  router.use(bodyParser.json())
+
+  router.post('/product', async (req: Request, res: Response) => {
+    const { title, body }: { title: string; body: string } = req.body
+    const installationToken: string = await app.app.getInstallationAccessToken({
+      installationId: parseInt(process.env.INSTALLATION_ID, 10)
+    })
+    const github = GitHubAPI({
+      auth: installationToken,
+      logger: app.log.child({ name: 'make-in-india-bot' })
+    })
```

Comment:
You can use Probot's method for returning an authenticated API client:

```suggestion
    const github = await app.auth(process.env.INSTALLATION_ID)
```

Reply:
Thanks @JasonEtco